kubectl-intermediate/step16.md

```diff
@@ -2,7 +2,7 @@ Patching can be used to partially update any kubernetes resources such as nodes,
 
 Let's create a pod with a label `env=prod` in the default namespace:
 
-`kubectl run nginx  --image=nginx --restart=Never --labels=env=prod`{{execute}}
```

Comment:
Why the removal of `--restart=Never`. I'm not aware of the history behind why it was added in the original version of the course.

Reply:
afaict the usage of --restart=Never used to be a common practice with earlier versions of k8s.  the main reason you see it removed is that, afaict, it serves no purpose.

Reply:
OK so it was not needed for the workshop to function. Thanks for confirming.